tests/irdl/test_declarative_assembly_format.py

```diff
@@ -2670,6 +2671,44 @@ class OptionalGroupOp(IRDLOperation):
     check_equivalence(program, generic_program, ctx)
 
 
+@pytest.mark.parametrize(
+    "program, generic_program",
+    [
+        (
+            '%0 = "test.op"() : () -> i32\ntest.optional_optional_group with "prop" %0',
```

Comment:
I believe you can omit the test.op and just have the `%0` be unbound

Reply:
Nope, tried it and it gave me an error.